src/bindings/crypto/bindings-napi.ts

```diff
@@ -0,0 +1,31 @@
+import { fieldsFromRustFlat, fieldsToRustFlat } from './bindings/conversion-base.js';
+
+export { bindingsNapi };
+
+function bindingsNapi(napi: any) {
+  return {
+    fp: {
+      vectorToRust: (fields: any) => {
+        console.log('values going in ', fields);
+        let res = fieldsToRustFlat(fields);
+        console.log('values going out ', res);
+        return res;
+      },
+      vectorFromRust: fieldsFromRustFlat,
```

Comment:
this just forwards to the "old" `fieldsFromRustFlat` and `fieldsToRustFlat` - both functions work fine for wasm and napi! but others will require a new implementation